src/main/java/org/rumbledb/cli/JsoniqQueryExecutor.java

```diff
@@ -194,30 +167,6 @@ public String runInteractive(java.nio.file.Path queryFile) throws IOException {
         return rddOutput;
     }
 
-    private JsoniqLexer getInputSource(String arg) throws IOException {
-        arg = arg.trim();
-        // return embedded file
```

Comment:
This was causing a lot of issues, it is now all handled in a unified way with the hadoop filesystem methods.